source/playbooks/CIS140/ssmdocs/CIS140_2.2.1.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { Construct } from 'constructs';
+import { PlaybookProps } from '../../SC/lib/control_runbooks-construct';
+import { ControlRunbookDocument } from '../../SC/ssmdocs/control_runbook';
+import { EnableEbsEncryptionByDefaultDocument } from '../../SC/ssmdocs/SC_EC2.7';
+
+export function createControlRunbook(stage: Construct, id: string, props: PlaybookProps): ControlRunbookDocument {
+  return new EnableEbsEncryptionByDefaultDocument(stage, id, { ...props, controlId: '2.2.1' });
```

Comment:
I think this still needs to be added to the file under `bin`. It's unfortunate that it's in so many places, but the bin file is what also adds an eventbridge rule for automated deployment to the admin stack